internal/rpc/jsonrpc/methods.go

```diff
@@ -4208,10 +4197,10 @@ func (s *Server) ticketInfo(ctx context.Context, icmd interface{}) (interface{},
 				BlockHeight: -1,
 				Status:      status.String(),
 			}
-			_, addrs, _, err := txscript.ExtractPkScriptAddrs(out.Version,
-				out.PkScript, w.ChainParams(), true) // Yes treasury
-			if err != nil {
-				return false, err
+
+			_, addrs := stdscript.ExtractAddrs(out.Version, out.PkScript, w.ChainParams())
+			if len(addrs) == 0 {
+				return false, errors.New("unable to decode ticket pkScript")
 			}
 			info.VotingAddress = addrs[0].String()
```

Comment:
In this case for example, it would panic if addrs was len 0, which could have previously happened with a nil error.